test/integration/image-component/default/pages/static-img.js

```diff
@@ -23,6 +23,12 @@ const Page = () => {
         layout="fixed"
         placeholder="blur"
       />
+      <Image
+        id="basic-static-raw"
```

Comment:
I'm not seeing a test with this id

Reply:
It's just the `'Should add a blur to a statically imported image in "raw" mode'` test. It's one of the tests that just looks at the html output so it doesn't actually use the id. I put the id in to help with manual debugging

Reply:
I see, so it might matcha different `<Image>` output but I think its fine for now